### DISCUSSION TITLE
Standalone codecs

### DISCUSSION THREAD

**Discussion by tansy** [cielavenir/p7zip#6] [General]
I thought about making building scripts for codecs as standalone plugins. The question is where to put these Makefiles? Where all the other 'bundles are (`$P7ZIP/CPP/7zip/Bundles`, `$P7ZIP/CPP/7zip/Bundles/Codecs`) or in Codecs directory (`$P7ZIP/Codecs`)? They Cannot be in `$P7ZIP/Codecs/codec` as they are 'taken' for codec sources and contain their own Makefiles.

**Comment by cielavenir** [comment]
I have already chosen CPP/7zip/Compress (Please don't bug me, Debian is already doing https://salsa.debian.org/debian/p7zip-rar/-/blob/debian/16.02-3/CPP/7zip/Compress/Rar/makefile?ref_type=tags)
